### ISSUE TITLE
Error when trying to wrap `VueDatePicker` component with it's own props type `VueDatePickerProps`

### ISSUE THREAD

**Issue by MellKam** [Vuepic/vue-datepicker#858]
In my project I created the wrapper component for datepicker with custom styles and exposed the original props by using `VueDatePickerProps` type. But after the update from `8.4.0` to `8.5.1` i got this error `Pre-transform error: Name in a signature must be an Identifier, ObjectPattern or ArrayPattern, instead got AssignmentPattern.`. Here is the simplified version of my component

```vue
<script setup lang="ts">
import Datepicker, {
	VueDatePickerProps,
	EmitEvents,
} from "@vuepic/vue-datepicker";
import "@vuepic/vue-datepicker/dist/main.css";
import { useEmitAsProps } from "radix-vue";

const props = withDefaults(defineProps<VueDatePickerProps>(), {
	config: () => ({ closeOnAutoApply: true }),
	autoApply: true,
	dark: true,
});

const emit = defineEmits<{
	[K in EmitEvents]: [...args: any[]];
}>();

const emitsAsProps = useEmitAsProps(emit);
</script>

<template>
	<Datepicker v-bind="{ ...props, ...emitsAsProps }" />
</template>

<style>
// custom styles...
</style>
```

Reproduction: https://stackblitz.com/edit/vuepic-vue-datepicker-2fraph?file=src%2Fcomponents%2FPlayground.vue

**Comment by peter-emad99** [comment]
i have the same error aslo

**Comment by Jasenkoo** [comment]
The suspected root cuse is the same as on #854

**Comment by Jasenkoo** [comment]
Confirmed, fixed in 441a8c67cfb9256db2da2b574e0560db5a59d8a1
